[Feature] Add annotation `@suppress` to AnnotationReader

I'm using Phan to statically analyze my code base.

It uses the annotation `@suppress` to suppress errors (ref.: https://github.com/phan/phan/wiki/Annotating-Your-Source-Code#suppress).

But the annotation `@suppress` is not read by Doctrine Annotations and I get the following error:

>  [ERROR] /Users/Aerendir/Documents/JooServer/Lab/Bundles/bundle-users/src/Command/AbstractUserCommand.php               
         Line: 23                                                                                                       
         Message: Could not parse class "SerendipityHQ\Bundle\UsersBundle\Command" for annotations. [Semantical Error]  
         The annotation "@suppress" in property SerendipityHQ\Bundle\UsersBundle\Command\AbstractUserCommand::$user
was never imported. Did you maybe forget to add a "use" statement for this annotation? 

I get it through `php-translation/symfony-bundle`, running the command `translation:extract` (opened issue: https://github.com/php-translation/symfony-bundle/issues/420).

In this moment is practically impossible for me to extract new versions using the bundle.

The fix is as easy as adding the annotation to the [`AnnotationReader::$globalIgnoredNames`](https://github.com/doctrine/annotations/blob/1.10.x/lib/Doctrine/Common/Annotations/AnnotationReader.php#L52).

--- a/lib/Doctrine/Common/Annotations/ImplicitlyIgnoredAnnotationNames.php
+++ b/lib/Doctrine/Common/Annotations/ImplicitlyIgnoredAnnotationNames.php
@@ -145,6 +145,8 @@ final class ImplicitlyIgnoredAnnotationNames
         'use' => true,
     ];
 
+    private const Phan = ['suppress' => true];
+
     public const LIST = self::Reserved
         + self::WidelyUsedNonStandard
         + self::PhpDocumentor1
@@ -157,7 +159,8 @@ final class ImplicitlyIgnoredAnnotationNames
         + self::Symfony
         + self::SlevomatCodingStandard
         + self::PhpCodeSniffer
-        + self::PhpStan;
+        + self::PhpStan
+        + self::Phan;
 
     private function __construct()
     {

